### ISSUE TITLE
Uncaught NL adapter error causes crash

### ISSUE THREAD

**Issue by jflasher** [openaq/openaq-fetch#490]
Not sure how often it happens, but just came across this.

```
/app/adapters/netherlands.js:113
      size: fp[3].trim()
                 ^

TypeError: Cannot read property 'trim' of undefined
    at Object.<anonymous> (/app/adapters/netherlands.js:94:13)
    at initialize.exports.each (/app/node_modules/cheerio/lib/api/traversing.js:293:24)
    at listApachetree (/app/adapters/netherlands.js:79:11)
    at Request._callback (/app/adapters/netherlands.js:22:20)
    at Request.self.callback (/app/node_modules/request/request.js:186:22)
    at emitTwo (events.js:125:13)
    at Request.emit (events.js:213:7)
    at Request.<anonymous> (/app/node_modules/request/request.js:1163:10)
    at emitOne (events.js:115:13)
    at Request.emit (events.js:210:7)
    at IncomingMessage.<anonymous> (/app/node_modules/request/request.js:1085:12)
    at Object.onceWrapper (events.js:314:30)
    at emitNone (events.js:110:20)
    at IncomingMessage.emit (events.js:207:7)
    at endReadableNT (_stream_readable.js:1045:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by dolugen** [comment]
Any way to reproduce this? (Did we have this conversation already? Or was this a similar bug?)

**Comment by jflasher** [comment]
I think it was probably just due to an unexpected page being scraped, easiest way to reproduce is probably just to set that var to undefined and make sure it errors without exiting. We've seen these types of things a lot, so it probably just *feels* like we had this conversation before :)

**Comment by majesticio** [comment]
fixed on commit 0ad6eb05b42b4fae25129b7c92f1bfd6aa1eb322
